Gemfile

```diff
@@ -81,6 +81,7 @@ gem "ransack", "~> 2.3" # Searching and sorting
 gem "recaptcha", "~> 5.5", require: "recaptcha/rails" # Helpers for the reCAPTCHA API
 gem "redcarpet", "~> 3.5" # A fast, safe and extensible Markdown to (X)HTML parser
 gem "redis", "~> 4.2.2" # Redis ruby client
+gem "redis-namespace", "~> 1.8" # Allows for namespacing in Redis
```

Comment:
The [blind pass through](https://github.com/resque/redis-namespace#blind-passthrough) issue worries me a little bit that it might cause some collisions between apps.

Reply:
I've used this gem in production before, and I'm not a big fan. We had lots of problems with things that aren't namespace aware, e.g. this as mentioned in the documentation:

> The effects of some redis commands cannot be limited to a particular namespace (e.g., `FLUSHALL`, which literally truncates all databases in your redis server, regardless of keyspace). 

This e.g. caused us to lose all background jobs once when we wanted to evict all keys from a specific cache. So the way to do this correctly is first looking for the correct keys yourself (with something like `KEYS  <namespace>-*`, then iterating over them and deleting them. This is an O(n) operation and was very slow for our use case (a heavily used cache with periodic full eviction).

Reply:
> The blind pass through issue worries me a little bit that it might cause some collisions between apps.

As Redis keeps improving each version, this might be a problem which would lock us on a specific version of Redis. Now we're only constrained by which versions the cloud providers support

I'm also quite frightened by:

> Redis::Namespace provides a namespaced interface to Redis by keeping an internal registry of the method signatures in Redis provided by the redis-rb gem; we keep track of which arguments need the namespace added, and which return values need the namespace removed.

It honestly feels like a tool useful to people that have small apps and want to separate different parts of it inside Redis without namespacing it themselves